pages/vi/profiles/kholood-ea.md

```diff
@@ -0,0 +1,23 @@
+ #  Kholood Eassa <br>
+ 
+ ![Egypt](https://media.webjet.com.au/Images/cities/1200x472/CAI1.jpg)
+
+Location/Time zone : Egypt, cairo, GMT+2 <br>
+OS: Mac High Sierra 10.13.6 <br>
+
+Software engineering graduate from king Abdulaziz university in Saudi Arabia,
+very passionate about software and technologies :)
+
+
+*Programming Languages*
++ Objective C
++ C#
++ C++
++Swift
+
+**Interested in**
+- Programming
+- Application Development
+- Problem Solving <br><br>
+
+<a href="https://www.linkedin.com/in/kholood-eassa-44220996//"> Linkde Account </a>
```

Comment:
Can you fix the spelling here. Also try not to use HTML link tag. There is Markdown for link as well.